addons/im_livechat/security/im_livechat_channel_security.xml

```diff
@@ -22,20 +22,20 @@
         </record>
 
     <data noupdate="1">
-        <record id="ir_rule_discuss_channel_group_im_livechat_group_manager" model="ir.rule">
-            <field name="name">discuss.channel: livechat manager can read all livechat channels</field>
+        <record id="ir_rule_discuss_channel_group_im_livechat_group_user" model="ir.rule">
```

Comment:
while we're renaming it, the first "group" is not useful as we're just repeating what comes after